File: backend/push-service/src/main/java/com/example/pushservice/fcm/dto/RequestChatDto.java
package com.example.pushservice.fcm.dto;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
public class RequestChatDto {
    private String chatRoomName;
    private String message;
    private String Image;
}

File: backend/push-service/src/main/kotlin/com/example/pushservice/fcm/dto/RequestChatDto.kt
package com.example.pushservice.fcm.dto

import com.fasterxml.jackson.annotation.JsonProperty

data class RequestChatDto(
    var chatRoomName: String? = null,
    var message: String? = null,
    /** 옛 자바 필드 이름이 대문자 `Image` 였다. JSON 키를 그대로 유지한다. */
    @field:JsonProperty("Image") var image: String? = null,
)
